openfermioncirq/gates/four_qubit_gates.py

```diff
@@ -63,29 +63,17 @@ def half_turns(self) -> Union[cirq.Symbol, float]:
         return self._exponent
 
     def _eigen_components(self):
+        minus_one_component = numpy.zeros((16, 16))
+        minus_one_component[3, 3] = minus_one_component[12, 12] = 0.5
+        minus_one_component[3, 12] = minus_one_component[12, 3] = 0.5
+
+        plus_one_component = numpy.zeros((16, 16))
+        plus_one_component[3, 3] = plus_one_component[12, 12] = 0.5
+        plus_one_component[3, 12] = plus_one_component[12, 3] = -0.5
+
```

Comment:
Use `numpy.ix_` for this. It makes it clearer that this is just an X gate on some subspace.

Reply:
I think this would be super nice for a gate operating on a larger subspace, but (as demonstrated by @babbush just now ;)) `ix_` is uncommon enough that we'd have to explain it a bit.